common/persistence/sql/sqlplugin/history_timer_tasks.go

```diff
@@ -51,11 +51,11 @@ type (
 	// TimerTasksFilter contains the column names within timer_tasks table that
 	// can be used to filter results through a WHERE clause
 	TimerTasksRangeFilter struct {
-		ShardID                int32
-		TaskID                 int64
-		MinVisibilityTimestamp time.Time
-		MaxVisibilityTimestamp time.Time
-		PageSize               int
+		ShardID                         int32
+		InclusiveMinTaskID              int64
+		InclusiveMinVisibilityTimestamp time.Time
+		ExclusiveMaxVisibilityTimestamp time.Time
```

Comment:
maybe we should rename task VisibilityTimestamp to FireTime.

Reply:
I thought about that before (), but then I saw the comments here specifically says those fields names are created to match the columns names in the underlying timer_tasks table. So I decide to leave them as is. 

Also if we were to make the change, we probably need to change `TimerTaskFilter` and `TimerTaskRow` to be consistent. But I don't think we can change the field in `TimerTaskRow`, otherwise the sql library will fail to populate the value. (Well, there're other ways around it like add additional field tags, but that probably doesn't worth it.)